Pully Penelope/Assets/Scripts/RoomTransfer.cs

```diff
@@ -18,10 +18,23 @@ private void Start()
     }
     private void OnTriggerEnter2D(Collider2D collision)
     {
-        if (collision.CompareTag("Player"))
+        /*if (collision.CompareTag("Player"))
         {
             cameraTransform.position += cameraChange;
+        }*/
+
+        if (collision.CompareTag("Player") || collision.CompareTag("Box"))
+        {
             collision.gameObject.transform.position += playerChange;
+            Debug.Log("Success");
+            if (collision.gameObject.GetComponent<HandleParent>().attachedObject != null)
+            {
+                collision.gameObject.GetComponent<HandleParent>().attachedObject.transform.position += playerChange;
+            }
+            if (collision.gameObject.CompareTag("Player") || collision.gameObject.GetComponent<HandleParent>().attachedObject.CompareTag("Player"))
```

Comment:
that's a lot of get components, would it be possible to add HandleParent as a variable to initialize in the script? if not, maybe you could trigger an event instead